File: Upcoder/Other/Liệt kê số chính phương nhỏ hơn/LTCB-120.py
import math

def check_scp(n):
    if n < 0:
        return False
    ck = int(math.sqrt(n))
    return ck * ck == n

n = int(input())
found = False

for i in range(1, n):
    if check_scp(i):
        print(i, end=" ")
        found = True

if not found:
    print(-1, end="")

File: Upcoder/Other/Liệt kê số chính phương nhỏ hơn/LTCB-120.cs
using System;
namespace SCP
{
    public class Program
    {
        static void Main(string[]args)
        {
            long n = long.Parse(Console.ReadLine());
            int count = 0;
            for(long i = 1; i < n; i++)
            {
                if(CheckSCP(i))
                {
                    Console.Write("{0} ", i);
                    count++;
                }
            }
            if(count == 0)
                Console.Write("-1");
        }
        static bool CheckSCP(long n)
        {
            if (n < 0) return false;
            long ck = (long)Math.Sqrt(n);
            return ck * ck == n;
        }
    }
}